docs/_includes/biblio.adoc

```diff
@@ -4,11 +4,49 @@ Included in:
 - user-manual: Bibliography
 ////
 
+Simple bibliographies are created by an unordered list with style `bibliography`.
+Cross references to bibliography entries are always rendered as [<label>], where <label> is the id of the reference, or the xreflabel if you gave it one.
+
+Example:
+
 .References
 ----
 include::ex-biblio.adoc[tags=base]
 ----
 
+Renders as:
+
 ====
 include::ex-biblio.adoc[tags=base]
 ====
+
+TIP: If you want advanced features such as auto-numbering and selectable citation styles, try the link:https://github.com/asciidoctor/asciidoctor-bibtex[asciidoctor-bibtex] project.
```

Comment:
FYI the `link:` prefix is implied when linking to a fully-qualified URL. So the macro can start with `https:`.